src/main/java/org/sagebionetworks/bridge/reporter/helper/BridgeHelper.java

```diff
@@ -98,6 +117,59 @@ public final void setBridgeClientManager(ClientManager bridgeClientManager) {
         return retList;
     }
     
+    public Map<StudyParticipant, RequestInfo> getRequestInfoForParticipant (String studyId, List<StudyParticipant> participants)
+            throws IOException {
+    	Map<StudyParticipant, RequestInfo> retMap = new HashMap<>();
+        
+        ForWorkersApi workersApi = bridgeClientManager.getClient(ForWorkersApi.class);
+        for (StudyParticipant participant : participants) {
+        	RequestInfo requestInfo = workersApi.getRequestInfoForWorker(studyId, participant.getId()).execute().body();
+    		retMap.put(participant, requestInfo);
+    		doSleep();
+        }
+        return retMap;
+    }
+    
+    public Map<RequestInfo, ActivityEventList> getActivityEventForParticipant (String studyId, List<StudyParticipant> participants)
+            throws IOException {
+    	Map<RequestInfo, ActivityEventList> retMap = new HashMap<>();
+        
+        ForWorkersApi workersApi = bridgeClientManager.getClient(ForWorkersApi.class);
+        for (StudyParticipant participant : participants) {
+        	RequestInfo requestInfo = workersApi.getRequestInfoForWorker(studyId, participant.getId()).execute().body();
+        	ActivityEventList activityEvents = workersApi.getActivityEventsForParticipantAndStudy(studyId, participant.getId()).execute().body();
+    		retMap.put(requestInfo, activityEvents);
+    		doSleep();
+        }
+        return retMap;
+    }
+    
+    public void uploadHealthData() throws IOException {
+    	String APP_VERSION = "version 1.0.0, build 2";
+        DateTimeZone CREATED_ON_TIMEZONE = DateTimeZone.forOffsetHours(9);
+        String PHONE_INFO = "Integration Tests";
+        String SCHEMA_ID = "health-data-integ-test-schema";
+        long SCHEMA_REV = 1L;
+        Map<String, String> data = ImmutableMap.<String, String>builder().put("answer-me", "C").build();
+        DateTime createdOn = DateTime.now(CREATED_ON_TIMEZONE);
+//        DateTime createdOn = new DateTime(2019, 8, 25, 9, 30);
+        HealthDataApi healthDataApi = bridgeClientManager.getClient(HealthDataApi.class);
+        
+        HealthDataSubmission submission = new HealthDataSubmission().appVersion(APP_VERSION).createdOn(createdOn)
+                .data(data).phoneInfo(PHONE_INFO).schemaId(SCHEMA_ID).schemaRevision(SCHEMA_REV);
+        
+        healthDataApi.submitHealthData(submission).execute().body();        
+    }
```

Comment:
This looks like test data in the BridgeHelper. This should be moved to a test class only in the test class path.